sympy/concrete/summations.py

```diff
@@ -180,11 +180,30 @@ def doit(self, **hints):
         else:
             f = self.function
 
+        # first make sure any definite limits have summation
+        # variables with matching assumptions
+        reps = {}
+        for xab in self.limits:
+            d = dummy_with_inherited_properties(xab)
+            if d:
+                reps[xab[0]] = d
+        if reps:
+            undo = dict([(v, k) for k, v in reps.items()])
```

Comment:
I like that variable name.

Reply:
You may probably even recognize it. :-)

Reply:
The beauty of forgetting...there are so many little things to see again, anew!

Anyway, I was making a mental not to start using this name after seeing this.